src/client/components/Footer/Footer.css

```diff
@@ -0,0 +1,109 @@
+body {
+    margin:0;
+    padding:0;
+    height:100%;       
+   }  
```

Comment:
YOu are adding styling to the body in the footer component. This could be **very** tricky to find! Please add it to some more general styling that affects the body